lib/modules/manager/bitrise/extract.ts

```diff
@@ -0,0 +1,38 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { parseSingleYaml } from '../../../util/yaml';
+import type { PackageDependency, PackageFileContent } from '../types';
+import { BitriseFile } from './schema';
+import { parseStep } from './utils';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string,
+): PackageFileContent | null {
+  const deps: PackageDependency[] = [];
+
+  try {
+    const parsed = parseSingleYaml(content, {
+      customSchema: BitriseFile,
+    });
+
+    const workflows = Object.values(parsed.workflows);
+    for (const workflow of workflows) {
+      const steps = workflow.steps.flatMap((step) => Object.keys(step));
+      for (const step of steps) {
+        const dep = parseStep(step, parsed.default_step_lib_source);
+
+        if (!is.nullOrUndefined(dep)) {
+          deps.push(dep);
+        }
+      }
+    }
+  } catch (err) {
+    logger.debug({ err, packageFile }, `Parsing Bitrise config YAML failed`);
```

Comment:
```suggestion
    logger.debug({ err, packageFile }, `Failed to parse Bitrise YAML config`);
```

Please check if this is the right place to make the change, before you apply it!

In any case, I would like the user-facing message to be: `Failed to parse Bitrise YAML config`. That way the important information is first, details second.